src/assets/i18n/hu.json

```diff
@@ -77,7 +77,7 @@
     "CONCERTS_CONTAINER__COMPONENT": {
       "CONCERTS__TITLE": "Koncertek",
       "PREVIOUS_CONCERTS__TITLE": "Korábbi koncertjeink",
-      "PREVIOUS_CONCERTS__ERROR": "A tartalom jelenleg nem elérhető.",
+      "ERROR_MESSAGE": "A tartalom jelenleg nem elérhető.",

```

Comment:
📓 itt lehet kicsit reszletesebben is, hogy az elozo koncertek nem erhetoek el jelenleg stb stb